src/main/java/com/adobe/commerce/cif/model/common/Attribute.java

```diff
@@ -28,7 +28,7 @@
     protected Object value;
     
     @ApiModelProperty(value = "If true, this attribute is a variant attribute. If not set or false, the attribute is a normal/simple attribute.")
-    protected Boolean variantAttribute;
+    protected Boolean variant;
```

Comment:
Maybe this could be little more specific instead of only `variant`? Variant only is not very self speaking here. Maybe `variantRelated`, `definesVariant` or something.

Reply:
Yes I also don't like it much but since I joined too late the wiki discussion I didn't want to complain after you all agreed! ;-) I cannot think of any better name ... something like `isVariantAxis` would be better, but because of some limitations in `jackson` the getter for Boolean would be `getIsVariantAxis()`, so a bit awkward, but still better. WDYT?

Reply:
`isVariantAxis` sounds good to me. Makes it also clear that it is a boolean.

Reply:
@mhaack ok for you?